elm/Swole/Components/MovementInput.elm

```diff
@@ -12,16 +12,15 @@ import Task
 import Dom exposing (focus)
 import List.Extra as List
 
-import Helpers.List as List
 import Helpers.Events exposing (onDeleteWhen)
 import Swole.Types.Movement exposing (Movement)
 
-type alias Movements = List (Int, Movement)
+type alias Movements = List Movement
 type alias Model = Movements
```

Comment:
I might have mentioned this previously - but does it make sense to flatten these lines to `type alias Model = List Movement`?

Reply:
I only did the `type alias Model` because I didn't want to expose the weird `Movements` type to the consuming component. BUT: last night I was thinking a lot about this and I'm going to rename this whole domain to `Complex` and create an associated type that this will work with directly, so all of this will go away at that point.